pkg/machines/libvirt-dbus.js

```diff
@@ -798,6 +800,26 @@ LIBVIRT_DBUS_PROVIDER = {
     },
 };
 
+export function deleteStorageVolume({
```

Comment:
This function is not needed. There is the storageVolumeDelete which was introduced some time ago which does the deletion, and we 'll need storagePooRefresh to refresh the pol.